.github/workflows/ci.yml

```diff
@@ -169,7 +171,7 @@ jobs:
       - name: Set up node
         uses: actions/setup-node@v2
         with:
-          node-version: 16.17.1
+          node-version: ${{ env.NODE_VERSION }}
```

Comment:
i saw this in another PR. Maybe base branch is incorrect?

Reply:
Yes, I'm waiting for the other to be merged.
This one isn't ready yet anyway :)